### ISSUE TITLE
Support metrics in the integrated dashboard

### ISSUE THREAD

**Issue by timoreimann** [digitalocean/DOKS#21]
We integrate the [upstream dashboard](https://github.com/kubernetes/dashboard). In order for metrics to show up on the dashboard, the dedicated `kubernetesui/metrics-scraper` sidecar needs to be integrated as well. This issue is to track the necessary work.

**Comment by TonyBogdanov** [comment]
@timoreimann Looks like you only use the repo for public issue tracking, is there anywhere else where we can check for progress, or do you handle that internally only? In other words, is anyone looking into it yet?

**Comment by timoreimann** [comment]
@TonyBogdanov we do maintain another, internal ticket. Work for the task at hand is planned, but we cannot commit to a specific timeline as of now.

The place to look for updates publicly is this issue. I'll make sure to post relevant updates here.

**Comment by TonyBogdanov** [comment]
@TonyBogdanov Awesome! Have a great day (or night) ;)

**Comment by ties-s** [comment]
Would really like this to work!

**Comment by nickmelis** [comment]
Any news on this?

**Comment by TonyBogdanov** [comment]
@nickmelis no and there probably won't be. Figure out something else :)

**Comment by ties-s** [comment]
> @nickmelis no and there probably won't be. Figure out something else :)

Is this based on any information?

**Comment by TonyBogdanov** [comment]
@ties-s No, only experience and the fact that there have been 2 major, stable versions of Kubernetes released on DO since the reporting of this issue and no work has been done on it as far as I know.

**Comment by VVVi** [comment]
Any progress here? Thanks.

**Comment by TonyBogdanov** [comment]
> Any progress here? Thanks.

Nope, just forget about it and figure out something else. I already said that 1 year ago.

**Comment by mkjmdski** [comment]
Is any work done on this? It doesn't make sense to use embedded dashboard if there's no way about getting pods metrics

**Comment by wscourge** [comment]
Shame

**Comment by MrCox007** [comment]
Any progress?

**Comment by ssuheesh** [comment]
Any progress?

**Comment by deoxyseia** [comment]
Any updates after 3 years?
